Resources/doc/definitions/type-inheritance.md

```diff
@@ -0,0 +1,101 @@
+Inheritance
+===========
+
+In some cases, inheritance can helps to don't repeat your self, beater organisation,
+clearer schema, easier interfaces implementation...
+
+Here an example using inheritance:
+
+```yaml
+ObjectB:
+    type: object
+    config:
+        interfaces: [InterfaceA]
+        fields:
+             foo: {type: String!}
+
+# ObjectA inherited config from InterfaceA and ObjectC
+ObjectA:
+    type: object
+    # ObjectB inherited config (fields, args...) from ObjectA
+    hiers: [ObjectB]
```

Comment:
I think you meant heirs? Like the person who inherits? If so I think this typo is system wide

Reply:
I think a good word for the configuration file could be `inherits` and internally when doing to inverse reference could indeed be `heirs`

Reply:
But I think you mean the inverse right? So just `heirs` could be good indeed

Reply:
Yes it `heirs` the correct word that is the inverse of `extends` right now.
You mean replace `extends` by `inherits`?